### PR TITLE
Add net.codelogistics.clicker

### PR DESCRIPTION
**Pull request by eyekay-gh**
### Please confirm your submission meets all the criteria

- [X] Please describe your application briefly.
- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. 
- [X] All assets referenced in the manifest are redistributable by any party. 
- [X] I am an author/developer/upstream contributor of the project.
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned.

Clicker is an auto clicker for X11 and Wayland, it can simulate user input like mouse clicks and key presses repeatedly at given intervals for a given duration.

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build net.codelogistics.clicker

**Comment by hfiguiere** [comment]
bot, build net.codelogistics.clicker

File: net.codelogistics.clicker.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://codeberg.org/eyekay/clicker.git",
+                    "tag": "0.1.5",
+                    "x-checker-data": {
+                        "type": "git",
+                        "tag-pattern": "^([\\d.]+)$"
+                    },
+                    "commit": "cc4cc727bca6fdbcc58c02e2260e8a5e4de4278f"
```
Comment (bbhtt) [inline net.codelogistics.clicker.json:76]:
```suggestion
                    "tag": "0.1.5",
                    "commit": "cc4cc727bca6fdbcc58c02e2260e8a5e4de4278f",
                    "x-checker-data": {
                        "type": "git",
                        "tag-pattern": "^([\\d.]+)$"
                    }
                    
```

Comment (bbhtt) [inline net.codelogistics.clicker.json:76]:
```suggestion
                    "tag": "0.1.5",
                    "commit": "cc4cc727bca6fdbcc58c02e2260e8a5e4de4278f",
                    "x-checker-data": {
                        "type": "git",
                        "tag-pattern": "^([\\d.]+)$"
                    }
```

File: net.codelogistics.clicker.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+		    "url": "https://files.pythonhosted.org/packages/b0/49/2fe589ce1fa6ca0f05ae0b1717923650f2cc6eec6307c71fbc7789738902/libevdev-0.11.tar.gz",
+		    "sha256": "e9ca006a4df2488a60bd9a740011ee948d81904be2364f017e560169508f560f",
+		    "x-checker-data": {
+                        "type": "pypi",
+                        "name": "libevdev"
+                    }
+		}
+	    ]
+	    },
+        {
```
Comment (bbhtt) [inline net.codelogistics.clicker.json:63]: Indentation is a bit broken here

Comment (eyekay-gh) [reply net.codelogistics.clicker.json]: Fixed.

File: net.codelogistics.clicker.json (hunk 1 above)
Comment (eyekay-gh) [reply net.codelogistics.clicker.json:76]: I've already updated this, I committed it while the bot was building the previous commit :facepalm:

Comment (bbhtt) [reply net.codelogistics.clicker.json]: The x-checker para should be at the end, the bot will reformat it on next update. This avoids that.

Currently you have `commit` as the last property.

File: net.codelogistics.clicker.json [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                    "type": "git",
+                    "url": "https://codeberg.org/eyekay/clicker.git",
+                    "tag": "0.1.5",
+		    "commit": "cc4cc727bca6fdbcc58c02e2260e8a5e4de4278f"
+                    "x-checker-data": {
+                        "type": "git",
+                        "tag-pattern": "^([\\d.]+)$"
+                    }
+                }
+            ]
```
Comment (bbhtt) [inline net.codelogistics.clicker.json:78]: Indentation is still a bit broken. Maybe something is wrong with your editor?

Comment (eyekay-gh) [reply net.codelogistics.clicker.json]: Hi, I've fixed it. I thought I could get away with using github's inbuilt editor, I have learnt my lesson now.

**Comment by bbhtt** [comment]
bot, build net.codelogistics.clicker

**Pull request closed without merging**